modules/developers-guide/examples/candid-ui.adoc

```diff
@@ -1,10 +1,10 @@
 [[candid-ui]]
 == Test functions in a browser
 
-The canister interface description language—often referred to as Candid or more generally as the IDL—provides a common language for specifying the signature of a canister service.
-Candid provides a unified way for you to interact with canisters that are written in different languages or accessed using different tools.
+The canister interface description language—often referred to as Candid or more generally as the IDL—provides a common language for specifying the signature of a canister smart contract.
```

Comment:
This is the only place where we could be using the term "service", since it maps to the Candid keyword "service" that I don't think we will change. That said, unless there is a specific link to the Candid terminology, your suggested change here is probably better.

Reply:
we do say "to share a common description for a service" 2 sentences later. so maybe it is then better to change it?
Not that I also did leave "service" in other places, but it was said on slack now that "it is ok to say a canister offers a service". Is that OK then?